src/components/screens/hwWalletLogin/selectAccount/accountCard.test.js

```diff
@@ -9,9 +9,11 @@ describe('Account Card', () => {
   beforeEach(() => {
     props = {
       account: {
-        address: '123456L',
-        balance: 100,
-        name: 'Lisk',
+        summary: {
+          address: '123456L',
```

Comment:
This is a legacy Lisk address.  Now that you've updated the mock value, you can update this. you can import an account from `test/constants` for ease.

Reply:
yes, we should update all fixtures to use correct values